packages/contracts/contracts/FeeDistributor.sol

```diff
@@ -106,9 +106,8 @@ contract FeeDistributor is CheckContract, FeeDistributorStorage, IFeeDistributor
         uint256 feeToSovCollector = toDistribute.mul(FEE_TO_SOV_COLLECTOR).div(
             LiquityMath.DECIMAL_PRECISION
         );
-        wrbtc.deposit{value: feeToSovCollector}();
-        wrbtc.approve(address(sovFeeCollector), feeToSovCollector);
-        sovFeeCollector.transferTokens(address(wrbtc), uint96(feeToSovCollector));
+
+        sovFeeCollector.transferRBTC{value: feeToSovCollector}();
```

Comment:
pls consider renaming `sovFeeCollector` to `feeSharingProxy` or to `sovFeeSharingProxy` although i would avoid using `sov` not to cause confusion with the SOV token.  
in the next lines `feeToSovCollector` can have different interpretations - fee convert to SOV and send to a collector?  
all in all i suggest to rename in all repos `feeSharingProxy` to `feeSharingCollector` which seems to be much more clear and unambiguous and use this name across all the repos.  
just to facilitate code perception and interpretation for future references.  
wdyt @ororopickpocket, @cowsant ?

Reply:
i like the suggestion to rename `feeSharingProxy` to `feeSharingCollector`. This is then also the name which should be used here.

Reply:
@cowsant pls address the conversation above

Reply:
This is already addressed in this commit @tjcloa  https://github.com/DistributedCollective/zero/pull/174/commits/30a0181f54940dcd91638aef8ce824e3964d6501